.github/ISSUE_TEMPLATE/bug-report-prebuilt-providers.md

```diff
@@ -0,0 +1,58 @@
+---
+name: "\U0001F41B Pre-built Provider Bug Report"
```

Comment:
Do we also need one for a pre-built provider request?

Reply:
If we want to endorse that at this point in time, we could add one. But I'd refrain from that until we fixed the scaling pains there.

Reply:
Yeah, let's not do that right now, especially since we just told Partner Alliances last week that we're not prepared to accommodate that